api-docs/generator.py

```diff
@@ -99,10 +86,20 @@ def init_db():
 
 def drop_db():
     report('db', 'Dropping database')
-    try:
-        os.remove(settings.DATABASES['default']['NAME'])
-    except (OSError, IOError):
-        pass
+    config = settings.DATABASES['default']
+    drop = Popen(
+        ['psql', '-U', config['USER'], '-h', config['HOST'],
+            '-c', 'DROP DATABASE {}'.format(config['NAME'])],
+        stdin=PIPE,
+        stdout=open(os.devnull, 'r+'))
+    drop.communicate()
```

Comment:
This is pretty gross, but I couldn't get django admin commands to drop the database, and using django's connections resulted in errors about not being able to drop the currently open database.